specification/cognitiveservices/data-plane/Language/preview/2021-05-01-preview/analyzeconversations.json

```diff
@@ -249,9 +255,29 @@
           "$ref": "#/definitions/AnalyzeParameters"
         }
       ],
+      "properties": {
+        "projectParameters": {
+          "type": "object",
+          "$ref": "#/definitions/DeepstackProjectParameters"
+        },
+        "callingOptions": {
+          "type": "object",
+          "$ref": "#/definitions/DeepstackCallingOptions"
+        }
+      }
+    },
+    "DeepstackProjectParameters": {
+      "properties": {
+        "query": {
```

Comment:
Given this is the only parameter, is this required? What about the projectParameters above? Please be sure to author `required` for both input and output models (for output models, it helps dictate the type declarations for the SDK).